jmx-metrics/docs/target-systems/hbase.md

```diff
@@ -0,0 +1,233 @@
+# Hbase Metrics
+
+The JMX Metric Gatherer provides built in Hbase metric gathering capabilities.
+These metrics are sourced from: https://hbase.apache.org/book.html#hbase_metrics
+
+### Metrics
+
+* Name: `hbase.master.region_server.count`
+* Description: The number of region servers.
+* Unit: `{servers}`
+* Labels: `state`
+* Instrument Type: longUpDownCounter
+
+
+* Name: `hbase.master.in_transition_regions.count`
+* Description: The number of regions that are in transition.
+* Unit: `1`
```

Comment:
should have units of {regions}. Same for next metric

Reply:
Updated